src/main/java/com/cgdpd/library/validation/Validator.java

```diff
@@ -33,6 +34,26 @@ public static String requiredValidIsbn(String paramName, String value) {
         return value;
     }
 
+    public static Optional<Long> checkTrackingStatus(String paramName,
+                                                     TrackingStatus trackingStatus,
+                                                     Optional<Long> value) {
```

Comment:
This method is way to specific to be in this class, let's move it to the BookCopyDTO, maybe in the future we can add something more generic